js/maps-setup.js

```diff
@@ -55,27 +61,39 @@ function initializeMap() {
                              // located at the URL which is given by the variable blueURL, see above
-              title: "first Marker",
-              window_content: "<h1>Marker1</h1><p> and this would be the extended description</p>"
+              title: "St. Thomas's Hospital",
+              window_content: "<h1>St. Thomas's Hopsital</h1><p> Initially dissolved in 1530, due to King Henry VIII's Act of Supremacy that confescated all church property, separating church and state. </p>"
              },
-             {position: new google.maps.LatLng(41.8902,12.4923),
+             {position: new google.maps.LatLng(51.516763,-0.098675),
               map: my_map,
               icon: blueURL, // this sets the image that represents the marker in the map
-              title: "second Marker",
-              window_content: "<h1>Marker2</h1><p> and <a href='http://something'>this would</a> be the extended description</p>"
+              title: "St. Bartholemew's Hospital",
+              window_content: "<h1>St. Bartholemew's Hospital</h1><p>One of the latter Royal Hospitals, located just North of the City's borders, here there began an emphasis on gardens, and outdoor space for patients</p>"
             },
-            {position: new google.maps.LatLng(41.8986,12.4768),
+            {position: new google.maps.LatLng(51.380922,-0.028995),
              map: my_map,
-             icon: redURL, // this sets the image that represents the marker in the map
-             title: "third Marker",
-             window_content: '<h1>Marker3</h1><img title="Picture of Quote. Src: someone, some year"  src="https://s-media-cache-ak0.pinimg.com/736x/6d/e2/25/6de2251b8b4be709dcc936ae4f0caaaf.jpg"/>' +
-             '<blockquote>quote quote quote quote</blockquote>'
-           }
-            ];
+             icon: blueURL, // this sets the image that represents the marker in the map
+             title: "Bethlem Hospital",
+             window_content: "<h1>Bethlem Hospital</h1><p></p>Known as 'Bedlam' Hospital during the time, one of the only hospitals in London to treat syphilis patients during the 17th Century. One of the first Royal Hopsitals reinstated after the dissolution of the Monasteries in 1530."
+           },
+           {position: new google.maps.LatLng(51.544976,-0.116053),
+             map: my_map,
+             icon: blueURL, // this sets the image that represents the marker in the map to the one
+                            // located at the URL which is given by the variable blueURL, see above
+             title: "BrideWell Hospital and Prison",
+             window_content: "<h1>BrideWell</h1><p> Both hospital and prison, solidified ideas of illness and evil and punishment for criminality or 'loose morals,' BridwWell is a testiment to the period's mentality on illness.</p>"
+            },
+            {position: new google.maps.LatLng(51.512344, -0.090985),
```

Comment:
ocnsider using a circle symbol here. We can discuss in class/with Amelia, who figured it out on her own.